modules/story-budget/story-budget.php

```diff
@@ -188,15 +188,20 @@ function register_term_columns() {
 	 */
 	function handle_form_date_range_change() {
 		
-		if ( !isset( $_POST['ef-story-budget-range-submit'], $_POST['ef-story-budget-number-days'], $_POST['ef-story-budget-start-date'] ) )
+		if ( !isset( $_POST['ef-story-budget-number-days'], $_POST['ef-story-budget-start-date'] ) &&
+		     ( !isset( $_POST['ef-story-budget-range-submit'] ) || !isset( $_POST['ef-story-budget-today-submit'] ) ) )
 			return;
 			
 		if ( !wp_verify_nonce( $_POST['nonce'], 'change-date' ) )
 			wp_die( $this->module->messages['nonce-failed'] );
 		
 		$current_user = wp_get_current_user();
 		$user_filters = $this->get_user_meta( $current_user->ID, self::usermeta_key_prefix . 'filters', true );
-		$user_filters['start_date'] = date( 'Y-m-d', strtotime( $_POST['ef-story-budget-start-date'] ) );
+		if ( isset( $_POST['ef-story-budget-today-submit'] ) ) {
+			$user_filters['start_date'] = date( 'Y-m-d' );
```

Comment:
What about using `current_time( 'Y-m-d' )` here? `date` wouldn't be accurate if the site's timezone was not UTC

Reply:
Or `date_i18n`

Reply:
👍  for `current_time`